ddtrace/tracer/stats.go

```diff
@@ -72,8 +74,8 @@ func newConcentrator(c *config, bucketSize int64) *concentrator {
 		Hostname:     c.hostname,
 		Env:          env,
 		Version:      c.version,
-		ContainerID:  "", //todo: verify these should be empty
-		GitCommitSha: "",
+		ContainerID:  "",
```

Comment:
nit: let's have a small comment here explaining why it's empty? (given this week's investigations on the feature, just want to be considerate of our future selves 😆 )